engine-rest/engine-rest-openapi/src/main/templates/paths/history/variable-instance/post.ftl

```diff
@@ -0,0 +1,103 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/variable-instance/post-variable-instance-query/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "queryHistoricVariableInstances"
+      tag = "Historic Variable Instance"
+      summary = "Get Variable Instances (POST)"
+      desc = "Queries for historic variable instances that fulfill the given parameters.
+              This method is slightly more powerful than the
+              [Get Variable Instances](${docsUrl}/reference/rest/history/variable-instance/get-variable-instance-query/)
+              method because it allows filtering by variable values of the different
+              types `String`, `Number` or `Boolean`."
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#assign requestMethod="POST"/>
+    <#include "/lib/commons/historic-variable-instance-query-params.ftl" >
+    <@lib.parameters
+        object = params
+        last = last
+    />
```

Comment:
Those are not supposed to be here, are they? They are part of the request body, right?